cita-executor/src/backlogs.rs

```diff
@@ -411,261 +411,261 @@ pub fn wrap_height(height: usize) -> u64 {
-        backlog.complete();
-    }
-
-    #[test]
-    fn test_complete_normal() {
-        let height = 9;
-        let open_block = generate_block(height);
-        let closed_block = generate_closed_block(open_block.clone());
-
-        let backlog = Backlog {
-            priority: Some(Priority::BlockWithProof),
-            open_block: Some(open_block),
-            proof: Some(generate_proof(height - 1)),
-            closed_block: Some(closed_block),
-        };
-        assert!(backlog.is_completed());
-        backlog.complete();
-    }
-
-    #[test]
-    fn test_backlogs_whole_flow() {
-        let open_block = generate_block(2);
-        let closed_block = generate_closed_block(open_block.clone());
-
-        // insert height 2 should be always failed
-        let mut backlogs = Backlogs::new(2, Default::default());
-        backlogs.insert_completed_result(1, generate_executed_result());
-        backlogs.insert_completed_result(2, generate_executed_result());
-        assert_eq!(
-            false,
-            backlogs.insert_open(
-                2,
-                Priority::Proposal,
-                open_block.clone(),
-                Some(generate_proof(1)),
-            ),
-            "insert staled block should return false"
-        );
-        assert_eq!(
-            false,
-            backlogs.insert_closed(2, closed_block),
-            "insert staled result should return false",
-        );
-        assert!(get_open_block(&backlogs, 2).is_none());
-
-        // insert height 3 should be ok
-        let open_block = generate_block(3);
-        let closed_block = generate_closed_block(open_block.clone());
-        assert_eq!(
-            true,
-            backlogs.insert_open(
-                3,
-                Priority::BlockWithProof,
-                open_block.clone(),
-                Some(generate_proof(2)),
-            ),
-        );
-        assert_eq!(
-            true,
-            backlogs.insert_closed(3, closed_block),
-            "insert current result should return true",
-        );
-        assert!(get_open_block(&backlogs, 3).is_some());
-
-        // complete height 3
-        assert!(backlogs.check_completed(3).is_ok());
-        let _backlog = backlogs.complete(3);
-        assert!(backlogs.get_completed_result(2).is_some());
-
-        assert!(get_open_block(&backlogs, 3).is_none());
-        assert_eq!(
-            false,
-            backlogs.insert_open(
-                3,
-                Priority::Proposal,
-                open_block.clone(),
-                Some(generate_proof(2)),
-            ),
-            "insert staled open_block should return false",
-        );
-    }
-
-    #[test]
-    fn test_wrap_height() {
-        assert_eq!(0, wrap_height(::std::usize::MAX));
-        assert_eq!(
-            ::std::usize::MAX as u64 - 1,
-            wrap_height(::std::usize::MAX - 1)
-        );
-        assert_eq!(2, wrap_height(2));
-    }
-}
+// #[cfg(test)]
```

Comment:
Do you run `cargo test` or `cargo test -p core?`

Maybe we should bring back the unit test into CI first, then we fix the unit test step by step. @leeyr338

Reply:
That's a good idea! Can you do this task? I am not familiar with CI.
BTW, I used the command
```shell
cargo test -p core-executor
```
to run Unit test for now.

Reply:
ok